src/app/settings/SettingsPage.tsx

```diff
@@ -56,7 +56,30 @@ const fakeWeapon = {
   },
   visible: true,
   primaryStat: {
-    value: 300,
+    value: 1600,
+  },
+  itemCategoryHashes: [],
+  destinyVersion: 2,
+};
+
+const fakeWeaponMasterwork = {
```

Comment:
`const fakeWeaponMasterwork = { ...fakeWeapon, isMasterwork: true };`

Reply:
Ahh see I knew there had to be a way to reuse that and just alter it. Thanks,